network-api/networkapi/templates/fragments/buyersguide/signup_requirement.html

```diff
@@ -3,6 +3,6 @@
 <div class="signup-requirement d-flex flex-column">
     <h4 class="pni-product-smaller-body-bold">{{ title }}</h4>
     <div class="explanation">
-        <p class="pni-product-smaller-body mb-0">{{ value | extended_yes_no }}</p>
+        <p class="pni-product-smaller-body mb-0">{{ value | extended_choice }}</p>
```

Comment:
While not directly related to the field `ai_is_untrustworthy`, this line has been updated to use the updated template tag mentioned [here](https://github.com/MozillaFoundation/foundation.mozilla.org/pull/12976/files#r1791075302).